src/main/java/fr/zcraft/quartzlib/tools/text/MessageSender.java

```diff
@@ -520,9 +520,17 @@ private static boolean sendChatPacket(Player receiver, String content, MessageTy
                     final Enum<?> nmsMessageType = type.getMessagePositionEnumValue();
 
                     if (nmsMessageType != null) {
-                        chatPacket = packetPlayOutChatClass
-                                .getConstructor(iChatBaseComponentClass, chatMessageTypeEnum)
-                                .newInstance(componentText, nmsMessageType);
+                        try {
+                            //Packet changed in 1.16+ an UUID is needed
+                            chatPacket = packetPlayOutChatClass
+                                    .getConstructor(iChatBaseComponentClass, chatMessageTypeEnum, UUID.class)
+                                    .newInstance(componentText, nmsMessageType, new UUID(0, 0));
```

Comment:
Adding a comment to explain why a null UUID work would be ideal.

Reply:
That's added, the main reason is with both long at 0 the client won't ignore the message, that was the old behaviour and if we allow this kind of message to be blocked that can lead to some issues